test/unit/test-url-replacements.js

```diff
@@ -1001,6 +1001,12 @@ describes.sandboxed('UrlReplacements', {}, (env) => {
         });
       });
 
+      it('should replace UACH', () => {
+        return expandUrlAsync('?sh=UACH(platform)').then((res) => {
+          expect(res).to.match(/sh=\w+/);
+        });
+      });
```

Comment:
optional:
```suggestion
      it('should replace UACH', async () => {
        await expect(expandUrlAsync('?sh=UACH(platform)').resolves.to.match(/sh=\w+/);
      });
```
may require tweaking

Reply:
How about making a separate PR to change the test style? Almost everything on that file could have been changed.

Reply:
SGTM; I don't feel strongly about this, I just try to point it out when I notice it because the `.resolves.to` is convenient syntactic sugar that not everyone has encountered.